concrete/js/blocks/google-map.js

```diff
@@ -0,0 +1,32 @@
+concreteGoogleMapInit = function () {
+  $('.googleMapCanvas').each(function () {
+    try {
+      var latitude = $(this).data('latitude');
+      var longitude = $(this).data('longitude');
+      var zoom = $(this).data('zoom');
+      var scrollwheel = $(this).data('scrollwheel');
+      var draggable = $(this).data('draggable');
+
+      var latlng = new google.maps.LatLng(latitude, longitude);
+
+      var mapOptions = {
+        zoom: zoom,
+        center: latlng,
+        mapTypeId: google.maps.MapTypeId.ROADMAP,
+        streetViewControl: false,
+        scrollwheel: scrollwheel,
+        draggable: draggable,
+        mapTypeControl: false
+      };
+
+      var map = new google.maps.Map(this, mapOptions);
+
+      new google.maps.Marker({
+        position: latlng,
+        map: map
+      });
+    } catch (e) {
+      $(this).replaceWith("<p>Unable to display map: " + e.message + "</p>")
```

Comment:
I believe this is a translatable string.

Reply:
Yes, I just copied it because I'm not sure how to fix this. The translation is made with PHP, but I need to replace a part of it with a JavaScript variable (`e.message`). I could do something like this `t('Unable to display map: ERR')` and then replace ERR in my JavaScript code, but that doesn't feel very nice. Do we even need this check? I don't think we have another block that displays error messages to the user like that. Any objection about removing it?

Reply:
:+1: about removing the English text: I'd write
```JavaScript
 $(this).replaceWith($('<p />').text(e.message));
```